quickstart.js

```diff
@@ -54,14 +54,15 @@ async function getServiceAccessToken() {
 
 async function getFromApi(serviceAccessToken, resourceUrl, queryParams = {}, raiseError = true) {
 
-  const url = `${baseUrl}${resourceUrl}`
+  const url = new URL(`${baseUrl}${resourceUrl}`);

```

Comment:
Thanks for fixing this. I think there's a slightly more straightforward way to do this:

```
async function getFromApi(serviceAccessToken, resourceUrl, queryParams = null, raiseError = true) {

  const queryString = queryParams ? `?${new URLSearchParams(queryParams)}` : '';
  const url = `${baseUrl}${resourceUrl}${queryString}`;

```

Reply:
updated